micro-rdk/src/common/camera/fake_camera.rs

```diff
@@ -57,6 +57,16 @@ impl Camera for FakeCamera {
             .map_err(|_| CameraError::CameraGenericError("failed to encode GetImageResponse"))?;
         Ok(buffer)
     }
+    fn render_frame(&mut self, mut buffer: BytesMut) -> Result<BytesMut, CameraError> {
+        let msg = google::api::HttpBody {
+            content_type: "image/jpeg".to_string(),
+            data: FAKE_JPEG.into(),
```

Comment:
is data a ref to an array or a vec?

Reply:
it's an owned vector, `pub data: ::prost::alloc::vec::Vec<u8>`. When removing `.into()`, compiler suggests `data: FAKE_JPEG.to_vec()`

Reply:
ok make sense